src/utils/RealtimeFeedUtils.js

```diff
@@ -165,7 +165,7 @@ function getDelayInformation(stopID: String, tripID: String, rtf: Object): ?Obje
   const info = rtf[tripID];
   let delay = parseInt(info.stopUpdates && info.stopUpdates[stopID]);
   if (Number.isNaN(delay)) delay = parseInt(info.delay);
-
+  if (delay <= 120) delay = 0;
   return {
     delay,
```

Comment:
you could remove the if statement above and just do `delay <= 120 ? 0 : delay` here

Reply:
wait @meganle i think we should keep it, and add a big comment directly above as to why we're doing this –– else it will be unclear to people reading in the future

Reply:
yeah that works! or we can make `120` into a constant called like `MIN_DELAY_BUFFER` or something more descriptive or similar since 120 is very arbitrary?